app/logic/emailHandler.py

```diff
@@ -1,7 +1,7 @@
 from datetime import datetime
 from peewee import DoesNotExist
 from flask_mail import Mail, Message
-from flask import g, session
+from flask import g, session, request

```

Comment:
None of these should be used in EmailHandler

Reply:
resolved